test/utils/block.ts

```diff
@@ -536,22 +536,33 @@ export const getEraIndexForBlock = async (api: ApiPromise, blockNumber: number):
     return eraAtBlock.unwrap().index.toNumber();
 };
 
+export const getBlockNumberAtWhichEraStarted = async (api: ApiPromise): Promise<number> => {
+    const chain = (await api.rpc.system.chain()).toString();
+    if (chain === "Stagelight") {
+        return 734672;
```

Comment:
maybe change it to constant like `const FIRST_ERA_WITH_PRESNT_VALIDATORS = 734672;` to make it clear what is this era.